.ci/scripts/calculate_jobs.py

```diff
@@ -134,11 +134,6 @@ def set_output(key: str, value: str):
                 "sytest-tag": "testing",
                 "postgres": "postgres",
             },
-            {
-                "sytest-tag": "buster",
```

Comment:
Do we also want to follow-up to remove `debian:buster` from the Docker builds on Sydent? [`.github/workflows/docker.yml#L28-L29`](https://github.com/matrix-org/sytest/blob/575569e3802462fd462cb47bc4ca1749fa922bc3/.github/workflows/docker.yml#L28-L29)

Reply:
Yes that was on my follow-up list :)